addons/udes_security/static/src/xml/custom_chatter_topbar.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="ChatterTopbarExtension" t-inherit="mail.ChatterTopbar" t-inherit-mode="extension" owl="1">
+        <xpath expr="//button[hasclass('o_ChatterTopbar_buttonScheduleActivity')]" position="replace">
+        <!-- Only show this button to admin users. -->
+            <t t-if="chatter.env.session.is_admin">
```

Comment:
Do we need to have both conditions on the t-if the previous one has_activities

Reply:
the target button we replace is already inside the `has_activities` t-if so both will be evaluated